### PR TITLE
help

### PR DESCRIPTION
**Pull request by jryanp1**
Please look at this

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
+const logger = ()=> {
+  const message = 'hi';
   console.log(message);
 };
 
-var looper = function() {
-  var message = 'I love JS!';
-  for (var i = 0; i < 10; i++) {
+const looper = () => {
+  } var message = 'I love JS!';
```
Comment (ryan-hamblin) [inline script.js:7]: Bracket out of place here

File: script.js [hunk 2]
```diff
@@ -1,21 +1,22 @@
-var logger = function() {
-  var message = 'hi';
+const logger = ()=> {
```
Comment (ryan-hamblin) [inline script.js:1]: add a space between your closing paren and your fat arrow.

File: script.js [hunk 3]
```diff
@@ -1,21 +1,22 @@
...
+  } var message = 'I love JS!';
+  for (let i = 0; i < 10; i++) {
     console.log(message);
   }
 };
 
-var greet = function(name) {
-  var greeting = 'Hello ' + name + '!';
+const greet = (name)=> {
+  var greeting = 'Hello my name is ${name}!';
```
Comment (ryan-hamblin) [inline script.js:14]: const and your quotes are not template strings. try using ' ` ' and not quotes.

**Review by ryan-hamblin — COMMENTED** [review]
Hey there, Looks like you have some very minor mistakes. Please button them up and commit your changes.

**Review by jryanp1 — COMMENTED** [review]
OK

**Pull request closed without merging**